app/models/tax_svc.rb

```diff
@@ -20,10 +31,19 @@ def get_tax(request_hash)
 
     if response['ResultCode'] != 'Success'
       logger.info_and_debug("Avatax Error: Order ##{order_number}", response)
-      raise 'error in Tax'
+
+      raise 'error in Tax' unless Spree::Config.avatax_address_validation
+      raise 'error in Tax' unless response['Messages'].any? do |message|
```

Comment:
Doesn't the avalara response have a better way of checking which error happened? Like a code or something instead of having to compare string messages?